src/screens/LoginScreen/LoginScreen.tsx

```diff
@@ -41,46 +47,67 @@ export default () => {
 		}
 	};
 
+	const handleForgotPassword = () => { console.log('Handle forgot password.'); };
+
 	return (
-		<View style={styles.outerContainer}>
-			<SpacerInline height={140} />
-			<Title text={`I am a ${userIdentity}.`} />
-			<SpacerInline height={40} />
-			<InputLabel text="Email Address" />
-			<TextInput
-				value={email}
-				onChangeText={setEmail}
-				style={styles.input}
-				autoCapitalize="none"
-				autoCorrect={false}
-				autoFocus={true}
-				blurOnSubmit={false}
-			/>
-			<InputLabel text="Password" />
-			<View style={styles.passwordContainer}>
-				<View style={{ flex: 8 }}>
-					<TextInput
-						textContentType="password"
+		<KeyboardAvoidingView style={styles.outerContainer} behavior="padding">
+			<View style={styles.banner}>
+				{/* TODO: banner should be a component */}
+				<Title text={`banana \n${userIdentity}`} />
+			</View>
+
+			<ScrollView style={styles.bodyContainer} contentContainerStyle={styles.bodyContentContainer}>
+				<View
+					style={styles.form}
+				>
+					<FormTextInput
+						label="email"
+						value={email}
+						setValue={setEmail}
+						style={styles.inputEmail}
+						placeholder="info@bananaapp.org"
+						autoCorrect={false}
+						autoCompleteType="username"
+						textContentType="username"
+						enablesReturnKeyAutomatically={true}
+						keyboardType="email-address"
+						returnKeyType="next"
+						blurOnSubmit={true} // Necessary to prevent focus from 'flickering'
+						onSubmitEditing={handleEmailInputSubmit}
+					/>
+
+					<FormTextInput
+						label="password"
 						value={password}
-						secureTextEntry={hidePwd}
-						onChangeText={setPassword}
-						style={styles.input}
-						autoCapitalize="none"
+						setValue={setPassword}
+						ref={passwordInputRef}
+						secureTextEntry={true}
 						autoCorrect={false}
+						autoCompleteType="password"
+						textContentType="password"
+						enablesReturnKeyAutomatically={true}
+						returnKeyType="go"
+						blurOnSubmit={false}
+						onSubmitEditing={handleLogin}
 					/>
+
+					<View style={styles.forgotPassword}>
+						{/* View wrapper required to constrain clickable area of button */}
+						<TouchableWithoutFeedback
+							onPress={handleForgotPassword}
+						>
+							<Text style={styles.forgotPasswordText}>
+								Forgot Password?
+							</Text>
+						</TouchableWithoutFeedback>
+					</View>
 				</View>
-				<View style={styles.iconContainer}>
-					<TouchableWithoutFeedback
-						onPress={() => setHidePwd(!hidePwd)}
-					>
-						<Icon name={hidePwd ? 'lock' : 'unlock'} style={styles.hideIcon} />
```

Comment:
I noticed you removed the Icon component. Don't we need that to display the eye icon and to give the user something to click on so they can view or hide their password?

Reply:
You're right. These commits were made when the final design came out, which had removed the lock icons. I raised your concern with Sangkee and she added the eyes.

This PR will be updated to match any other changes since the final design